game/src/ui/gameplay/BuildTool/GoldCost.gd

```diff
@@ -0,0 +1,16 @@
+extends Node2D
```

Comment:
Whats this file here for, can we delete? Its not even referenced in GoldCost.tscn